sympy/printing/tests/test_latex.py

```diff
@@ -2249,6 +2250,45 @@ def test_KroneckerProduct_printing():
     assert latex(KroneckerProduct(A, B)) == r'A \otimes B'
 
 
+def test_Series_printing():
+    tf1 = TransferFunction(x*y**2 - z, y**3 - t**3, y)
+    tf2 = TransferFunction(x - y, x + y, y)
+    tf3 = TransferFunction(t*x**2 - t**w*x + w, t - y, y)
+    assert latex(Series(tf1, tf2)) == \
+        '\\frac{\\left(x - y\\right) \\left(x y^{2} - z\\right)}{\\left(- t^{3} + y^{3}\\right) \\left(x + y\\right)}'
+    assert latex(Series(tf1, tf2, tf3)) == \
+        '\\frac{\\left(x - y\\right) \\left(x y^{2} - z\\right) \\left(t x^{2} - t^{w} x + w\\right)}{\\left(t - y\\right) \\left(- t^{3} + y^{3}\\right) \\left(x + y\\right)}'
+    assert latex(Series(-tf2, tf1)) == \
+        '\\frac{\\left(- x + y\\right) \\left(x y^{2} - z\\right)}{\\left(- t^{3} + y^{3}\\right) \\left(x + y\\right)}'
+
+
+def test_TransferFunction_printing():
+    tf1 = TransferFunction(x - 1, x + 1, x)
+    assert latex(tf1) == r"\frac{x - 1}{x + 1}"
+    tf2 = TransferFunction(x + 1, 2 - y, x)
+    assert latex(tf2) == r"\frac{x + 1}{2 - y}"
+    tf3 = TransferFunction(y, y**2 + 2*y + 3, y)
+    assert latex(tf3) == r"\frac{y}{y^{2} + 2 y + 3}"
+
+
+def test_Parallel_printing():
+    tf1 = TransferFunction(x*y**2 - z, y**3 - t**3, y)
+    tf2 = TransferFunction(x - y, x + y, y)
+    assert latex(Parallel(tf1, tf2)) == \
+        '\\frac{\\left(- t^{3} + y^{3}\\right) \\left(x - y\\right) + \\left(x + y\\right) \\left(x y^{2} - z\\right)}{\\left(- t^{3} + y^{3}\\right) \\left(x + y\\right)}'
+    assert latex(Parallel(-tf2, tf1)) == \
+        '\\frac{\\left(- t^{3} + y^{3}\\right) \\left(- x + y\\right) + \\left(x + y\\right) \\left(x y^{2} - z\\right)}{\\left(- t^{3} + y^{3}\\right) \\left(x + y\\right)}'
+
+
+def test_Feedback_printing():
+    tf1 = TransferFunction(p, p + x, p)
+    tf2 = TransferFunction(-s + p, p + s, p)
+    assert latex(Feedback(tf1, tf2)) == \
+        '\\frac{p \\left(p + s\\right)}{p \\left(p - s\\right) + \\left(p + s\\right) \\left(p + x\\right)}'
+    assert latex(Feedback(tf1*tf2, TransferFunction(1, 1, p))) == \
+        '\\frac{p \\left(p - s\\right)}{p \\left(p - s\\right) + \\left(p + s\\right) \\left(p + x\\right)}'
```

Comment:
These should follow the same recommendations I made in the pretty print tests to better follow the unevaluated form of these three classes.